creation/web_base/setup_x509.sh

```diff
@@ -200,24 +200,30 @@ copy_x509_proxy() {
     return 0
 }
 
+
 config_idtoken() {
     #
-    if [ -e "$GLIDEIN_CONDOR_TOKEN" ]; then
-        export GLIDEIN_CONDOR_TOKEN_ORIG="$GLIDEIN_CONDOR_TOKEN"
-        idtoken=$(basename $GLIDEIN_CONDOR_TOKEN)
-        local_proxy_dir=$(pwd)/ticket
-        mkdir -p $local_proxy_dir
-        cp $GLIDEIN_CONDOR_TOKEN_ORIG $local_proxy_dir/$idtoken
-        export GLIDEIN_CONDOR_TOKEN=$local_proxy_dir/$idtoken
-        echo "glidein IDTOKEN set to $GLIDEIN_CONDOR_TOKEN"
-        if [ ! -e "$GLIDEIN_CONDOR_TOKEN" ]; then
-            "$error_gen" -error "setup_x509.sh" "Corruption" "IDTOKEN" "$GLIDEIN_CONDOR_TOKEN" "not_copied"
+    if [ -e "${GLIDEIN_CONDOR_TOKEN}" ]; then
+        export GLIDEIN_CONDOR_TOKEN_ORIG="${GLIDEIN_CONDOR_TOKEN}"
+        idtoken="$(basename "${GLIDEIN_CONDOR_TOKEN}")"
+        cred_base="$(dirname "${GLIDEIN_CONDOR_TOKEN}")"
+        local_proxy_dir="$(pwd)/ticket"
+        mkdir -p "${local_proxy_dir}"
+        for TOK in ${cred_base}/*.idtoken; do
```

Comment:
If GLIDEIN_CONDOR_TOKEN is not defined (old frontend not sending token) also other tokens are not copied